packages/common/src/lib/common/tools.ts

```diff
@@ -2359,20 +2407,18 @@ function encrypt(key, value) {
 /**
  * encrypts a value by a given key via AES-192-CBC
  *
- * @param {string} key - Secret key
- * @param {string} value - value to decrypt
- * @returns {string}
+ * @param key - Secret key
+ * @param value - value to decrypt
  */
-function decrypt(key, value) {
+function decrypt(key: string, value: string): string {
     // if not encrypted as aes-192 or key not a valid 48 digit hex -> fallback
     if (!value.startsWith(`$/aes-192-cbc:`) || !/^[0-9a-f]{48}$/.test(key)) {
         return decryptLegacy(key, value);
     }
 
-    crypto = crypto || require('crypto');
-
-    const textParts = value.split(':');
+    const textParts: string[] = value.split(':');
     const iv = Buffer.from(textParts[1], 'hex');
+    /** @ts-expect-error ts doesn't know if an element is in */
     const encryptedText = Buffer.from(textParts.pop(), 'hex');
```

Comment:
I think this captures the intent better (if I didn't misunderstand) and also documents what the parts are.
```suggestion
    const textParts = value.split(':', 3) as [alg: string, iv: string, plaintext: string];
    const iv = Buffer.from(textParts[1], 'hex');
    const encryptedText = Buffer.from(textParts[2], 'hex');
```